### PR TITLE
try building on OSX

### PR DESCRIPTION
**Pull request by dpryan79**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pytabix/build.sh [hunk 1]
```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-
-export CFLAGS="-I$PREFIX/include"
-export LDFLAGS="-L$PREFIX/lib"
-export LD_LIBRARY_PATH="-L$PREFIX/lib"
-export CPATH=${PREFIX}/include
```
Comment (mbargull) [inline recipes/pytabix/build.sh]: The compiler packages should set this up for us now.

Comment (dpryan79) [reply recipes/pytabix/build.sh]: Thanks @mbargull that helped clean things up :)

**Comment by mbargull** [comment]
BTW, should I rather "squash & merge" or let the bot do its thing?

**Comment by dpryan79** [comment]
If possible, just have the bot do its thing. That doesn't work 100% of the time (the bot will post that it failed if there's a problem), but this tends to circumvent the sporadic networking errors we've all seen when uploading from circleci (it can also circumvent needing to build in the master branch, but given all of the caching that goes on I'm not sure how much time we save there in practice).

**Comment by mbargull** [comment]
Got it, makes sense, thank you!

**Pull request merged**